tests/integrations/bottle/test_bottle.py

```diff
@@ -234,9 +234,7 @@ def index():
     assert response[1] == "200 OK"
 
     (event,) = events
-    assert event["_meta"]["request"]["data"] == {
-        "": {"len": 2000, "rem": [["!config", "x", 0, 2000]]}
-    }
+    assert event["_meta"]["request"]["data"] == {"": {"rem": [["!config", "x"]]}}
```

Comment:
did you check that ingestion does not fail with sending less fields than before?

Reply:
Yes. Joris said here that those fields are optional: https://github.com/getsentry/sentry-python/issues/1646 (see the slack screenshots)

Reply:
Ok, I did not check myself. I trust Joris...